src/org/folio/rest_v2/Eureka.groovy

```diff
@@ -0,0 +1,86 @@
+package org.folio.rest_v2
+
+import org.folio.models.EurekaTenant
+import org.folio.utilities.RequestException
+
+class Eureka extends Authorization {
```

Comment:
Do we really need to extends from Authorization class or better from Common?

Reply:
@OHaimanov 
I took an example from **OkapiTenant**.
So do you suggest to extend from **Common** class directly?

Reply:
Yes, I think it is an overhead to extend Auth calss, as Eureka will have own auth related methods, I think maybe after final version we will do some optimisations, but for now it is ok